src/gitpod.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from './component';
+import { NodeProject } from './node-project';
+import { YamlFile, YamlFileOptions } from './yaml';
+
+const GITPOD_FILE = '.gitpod.yml';
+
+export interface GitpodOptions extends YamlFileOptions {
+
+}
```

Comment:
Not sure I like this leakage. I rather we under-model this based on the current usage and extend in the future instead of just leak the entire file as an API. That's the whole point of projen eventually..

```suggestion
export interface GitpodOptions {
  readonly init?: string;
  readonly command?: string;
}
```

Reply:
why is JsonFile different which is leaked

Reply:
It can't be that simple either.  The .gitpod.yml can be much longer

Reply:
`JsonFile` mental model is about JSON files, and therefore it's not an abstraction leak. The API for `Gitpod` should "speak" in gitpod terms.

I realize it can't be that simple as that but we can definitely start with that (or more if you'd like) and evolve it.

Reply:
fair enough